File: code1/strRev.py
''' ======= Code To Express ======= '''
''' ========= Daily Codes ========= '''

def strRev(a):
    return a[::-1]

stringList=['Hello','Greetings']
stringRevList=[strRev(i) for i in stringList]

for i in stringRevList:
    print(i)

File: legacy/code2/strRev.js
/* ===== The Lean Programmer ===== */
/* ========= Daily Codes ========= */

function dailyCode (str) {
    return str
      .split('')
      .reverse()
      .join('');
}

// Guess the output!
console.log(dailyCode('Hello')); 
console.log(dailyCode('Greetings')); 